cypress/fixtures/ledgerActions/ethereum-call-contract.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "ethereum-call-contract",
+  "payload": {
+    "contract_address": "ethereum_address",
+    "data": "some_bytes",
+    "gas_limit": "10000000000",
+    "network": "1",
+    "min_block_timestamp": ""
```

Comment:
❓ isn't it `null` or no line when it's not set by comit_node?

Reply:
At the moment we serialize to `null` I think.